code/modules/client/preferences.dm

```diff
@@ -264,6 +264,16 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 
 				mutant_colors = TRUE
 
+			if((DYNCOLORS in pref_species.species_traits))
+
+				if(!use_skintones)
+					dat += APPEARANCE_CATEGORY_COLUMN
+
+				dat += "<h3>Ethereal Color</h3>"
+
+				dat += "<span style='border: 1px solid #161616; background-color: #[features["ethcolor"]];'>&nbsp;&nbsp;&nbsp;</span> <a href='?_src_=prefs;preference=color_ethereal;task=input'>Change</a><BR>"

```

Comment:
could you possibly make this more generic if it's not too much trouble? Other races might be able to use something like this (flypeople, moths)

Reply:
mcolor

Reply:
isn't mcolor picking a color directly from the full palette?

Reply:
Maybe I'm reading it wrong but this is in
`if((DYNCOLORS in pref_species.species_traits))`
so if I set another race with DYNCOLORS I'd be getting the "Ethereal Colors" menu

What I meant was having the same thing except being able to pick from a different list depending on the race

Reply:
i can look into that potentialy